docs/otoc/otoc_example.ipynb

```diff
@@ -0,0 +1,761 @@
+    "        if the notebook is executed in Colab. In case the notebook is executed in Jupyter notebook\n",
+    "        or other IPython runtimes, no interactive login is provided, it is assumed that the \n",
+    "        `GOOGLE_APPLICATION_CREDENTIALS` env var is set or `gcloud auth application-default login`\n",
+    "        was executed already.\n",
+    "\n",
+    "        For more information on using Application Default Credentials see \n",
+    "        https://cloud.google.com/docs/authentication/production\n",
+    "        \"\"\"\n",
+    "        in_colab = False\n",
+    "        try:\n",
+    "            from IPython import get_ipython\n",
+    "            in_colab = 'google.colab' in str(get_ipython())\n",
+    "        except: \n",
+    "            # Notebook is not executed within IPython. Assuming external authentication.\n",
+    "            return \n",
+    "\n",
+    "        if in_colab: \n",
+    "            from google.colab import auth      \n",
+    "            print(\"Getting OAuth2 credentials.\")\n",
+    "            print(\"Press enter after entering the verification code.\")\n",
+    "            auth.authenticate_user(clear_output=False)\n",
+    "            print(\"Authentication complete.\")\n",
+    "        else: \n",
+    "            print(\"Notebook is not executed with Colab, assuming Application Default Credentials are setup.\") \n",
+    "\n",
+    "    authenticate_user()\n",
+    "    print(\"Successful authentication to Google Cloud.\")\n",
+    "\n",
+    "processor_id = \"\" #@param {type:\"string\"}\n",
+    "if use_noisy_simulator or processor_id == \"\":\n",
+    "    print(\"Using a noisy simulator.\")\n",
+    "    sampler = cg.PhasedFSimEngineSimulator.create_with_random_gaussian_sqrt_iswap(\n",
+    "        mean=cg.SQRT_ISWAP_PARAMETERS,\n",
+    "        sigma=cg.PhasedFSimCharacterization(\n",
+    "            theta=0.01, zeta=0.10, chi=0.01, gamma=0.10, phi=0.02\n",
+    "        ),\n",
+    "    )\n",
+    "    device = cg.Bristlecone\n",
+    "else:\n",
+    "    sampler = cg.get_engine_sampler(processor_id, gate_set_name=\"sqrt_iswap\")\n",
+    "    device = cg.get_engine_device(processor_id=processor_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "yLqyFVH2zn8D"
+   },
+   "source": [
+    "## Build the OTOC circuits"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "pb8GcwQ2As7q"
+   },
+   "source": [
+    "### Picking qubits"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "2DbErcZ_FpKX"
+   },
+   "source": [
+    "First we specify a line of qubits to use. If running on the `cirq_google.Engine`, use the latest calibration report to identify a good line of qubits."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "id": "GZIsJoNX1omO"
+   },
+   "outputs": [],
+   "source": [
+    "if not use_noisy_simulator:\n",
```

Comment:
`or processor_id == ""`

[or fix it so `use_noisy_simulator` is True if processor_id == ""]. (non optional)

Reply:
Fixed so `use_noisy_simulator` is True if `processor_id == ""`